### PR TITLE
Renaming batch_lengts back to lens, disabling tests for now

### PR DESCRIPTION
**Pull request by borisfom**
# What does this PR do ?

Add a one line overview of what this PR aims to accomplish.

**Collection**: [Note which collection this PR will affect]

# Changelog 
- Add specific line by line info of high level changes in this PR.

# Usage
* You can potentially add a usage example below

```python
# Add a code snippet demonstrating how to use this 
```

# Before your PR is "Ready for review"
**Pre checks**:
- [ ] Make sure you read and followed [Contributor guidelines](https://github.com/NVIDIA/NeMo/blob/main/CONTRIBUTING.md)
- [ ] Did you write any new necessary tests?
- [ ] Did you add or update any necessary documentation?
- [ ] Does the PR affect components that are optional to install? (Ex: Numba, Pynini, Apex etc)
  - [ ] Reviewer: Does the PR have correct import guards for all optional libraries?
  
**PR Type**:
- [ ] New Feature
- [ ] Bugfix
- [ ] Documentation

If you haven't finished some of the above items you can still open "Draft" PR.

## Who can review?

Anyone in the community is free to review the PR once the checks have passed. 
[Contributor guidelines](https://github.com/NVIDIA/NeMo/blob/main/CONTRIBUTING.md) contains specific people who can review PRs to various areas.

# Additional Information
* Related to # (issue)

### CODE REVIEW

File: nemo/collections/tts/helpers/helpers.py [hunk 1]
```diff
@@ -756,7 +756,7 @@ def sample_tts_input(
     sz = (max_batch * max_dim,) if export_config["enable_ragged_batches"] else (max_batch, max_dim)
     inp = torch.randint(*export_config["emb_range"], sz, device=device, dtype=torch.int64)
     pitch = torch.randn(sz, device=device, dtype=torch.float32) * 0.5
-    pace = torch.clamp(torch.randn(sz, device=device, dtype=torch.float32) * 0.1 + 1, min=0.01)
+    pace = torch.clamp(torch.randn(sz, device=device, dtype=torch.float32) * 0.1 + 0.7, min=0.2)
```
Comment (blisc) [inline nemo/collections/tts/helpers/helpers.py:759]: Can you default this back to 1?

File: nemo/collections/tts/models/radtts.py [hunk 2]
```diff
@@ -436,7 +436,7 @@ def _prepare_for_export(self, **kwargs):
         # Define input_types and output_types as required by export()
         self._input_types = {
             "text": NeuralType(tensor_shape, TokenIndex()),
-            "batch_lengths": NeuralType(('B')),
+            "lens": NeuralType(('B')),
```
Comment (blisc) [inline nemo/collections/tts/models/radtts.py:439]: Let's keep batch_lengths for now

File: tests/collections/tts/test_tts_exportables.py [hunk 3]
```diff
@@ -78,6 +78,7 @@ def test_HifiGanModel_export_to_onnx(self, hifigan_model):
             filename = os.path.join(tmpdir, 'hfg.onnx')
             model.export(output=filename, verbose=True, check_trace=True)
 
+    @pytest.mark.pleasefixme
```
Comment (blisc) [inline tests/collections/tts/test_tts_exportables.py:81]: I thought torchscript was fine. It was only onnx failing

Comment (borisfom) [reply tests/collections/tts/test_tts_exportables.py:81]: No they both fail in Torch (not just in exported code).

Comment (borisfom) [reply tests/collections/tts/test_tts_exportables.py:81]: Superseded by https://github.com/NVIDIA/NeMo/pull/6036

**Pull request closed without merging**